creativecloud/blocks/nonprofit/constants.js

```diff
@@ -0,0 +1,427 @@
+// eslint-disable-next-line import/prefer-default-export
+export const countries = [
+  {
+    code: 'AFG',
+    name: 'Afghanistan',
+  },
```

Comment:
As the country names are in code, I believe we won't be able to localize it. Could you please confirm if this block is for en language only?

Reply:
I see. 
Well I think we should still merge it since there's a few more items will open PRs for. The important thing is we get this merged so GWP can start building the pages. Nothing in the PR will go live until mid November. However web publishing was meant to start Sept. 30th but has delayed for various reasons out of our control. Thank you